include/wamr/WAMRWasmModule.h

```diff
@@ -42,7 +43,11 @@ namespace wasm {
     private:
         bool _isBound;
 
-        std::vector<char> errorBuffer;
+        char errorBuffer[ERROR_BUFFER_SIZE];
+
+#if(WAMR_EXECUTION_MODE_INTERP)
+        std::vector<uint8_t> wasmBytes;
```

Comment:
Why does this need to be a property of the class? AFAICT it's only used in one method (`bindToFunction`), so it can just be a local variable in that method.

Reply:
Unfortunately not, because wasm_runtime_load creates the WASMModule but doesn't copy the code into this module, hence you have to keep the code somewhere else during the whole execution. 
This solved the bug (I mentioned on slack) that the first execution works fine in interpreter but the next ones will fail.